docs/docs/cmd/teams/team/team-add.md

```diff
@@ -55,3 +55,46 @@ m365 teams team add --name "Architecture" --description "Architecture Discussion
 - Get started with Teams templates: [https://docs.microsoft.com/MicrosoftTeams/get-started-with-teams-templates](https://docs.microsoft.com/MicrosoftTeams/get-started-with-teams-templates)
 - group resource type: [https://docs.microsoft.com/graph/api/resources/group?view=graph-rest-1.0](https://docs.microsoft.com/graph/api/resources/group?view=graph-rest-1.0)
 - teamsAsyncOperation resource type: [https://docs.microsoft.com/graph/api/resources/teamsasyncoperation?view=graph-rest-1.0](https://docs.microsoft.com/graph/api/resources/teamsasyncoperation?view=graph-rest-1.0)
+
+## Response
```

Comment:
Lets add an extra response header when users make use of the `--wait` option.

![afbeelding](https://user-images.githubusercontent.com/38426621/199959925-d242dadc-23b4-48c2-9015-6964a9ce2176.png)